technical-reports/format/terminology.md

```diff
@@ -26,34 +26,46 @@ For example:
 
 ## <dfn>Design tool</dfn>
 
-Visual design creation and editing tools.
+A Design tool is a tool for visual design creation and editing.
 
-This includes:
+For example:
 
 - Bitmap image manipulation programs:
-  - Photoshop
-  - Affinity Photo
-  - Paint.net
+  - [Photoshop](https://www.adobe.com/products/photoshop.html)
+  - [Affinity Photo](https://affinity.serif.com/photo)
+  - [Paint.net](https://www.getpaint.net/)
 - Vector graphics tools:
-  - Illustrator
-  - Inkscape
-- UI design and prototyping tools:
-  - Adobe XD
-  - UXPin
-  - Sketch
-  - Figma
+  - [Illustrator](https://www.adobe.com/products/illustrator.html)
+  - [Inkscape](https://inkscape.org/)
+- UI design, wireframing and prototyping tools:
+  - [Adobe XD](https://www.adobe.com/products/xd.html)
+  - [UXPin](https://www.uxpin.com/)
+  - [Sketch](https://www.sketch.com/)
+  - [Figma]https://www.figma.com/()
   - ...
 
 ## <dfn>Translation tool</dfn>
 
-Token translation tools are tools that translate token data from one format to another.
+A token's translation tool is a tool that translates token data from one format to another.
 
 For example:
 
 - [Theo](https://github.com/salesforce-ux/theo)
 - [Style Dictionary](https://amzn.github.io/style-dictionary/)
 - [Diez](https://diez.org/)
 - [Specify](https://specifyapp.com/)
+- ...
+
+## <dfn>Documentation tool</dfn>
+
+A documentation tool is a tool for documenting design tokens usage.
+
+For example:
+
+- [Storybook](https://github.com/salesforce-ux/theo)
+- [Zeroheight](https://amzn.github.io/style-dictionary/)
+- [Backlight](https://backlight.dev/)
```

Comment:
Would Specify, Supernova, and Knapsack also belong to this category?

Reply:
Yes you're right :)
Just added them!